CHANGELOG.md

```diff
@@ -3,6 +3,61 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.14.0
+
+_Aug 25, 2022_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Introduce column grouping for data grid (#5133) @alexfauquette
+
+  You can now group columns using the `columnGroupingModel` prop. This lets you to display more structured data.
+
+  ![image](https://user-images.githubusercontent.com/45398769/186178366-4fba66b2-bf90-4c7a-9d83-940a7fc78704.png)
+
+  To enable this feature, add `experimentalFeatures={{ columnGrouping: true }}`.
+  The grouping header can be fully customized.
+  See the [documentation](https://mui.com/x/react-data-grid/column-groups/) to explore everything it has to offer.
+
+- 🧪 Add new unstable [`DateField`](https://mui.com/x/react-date-pickers/date-field/) and [`DateRangeField`](https://mui.com/x/react-date-pickers/date-range-field/) components for date pickers (#5504) @flaviendelangle
```

Comment:
@flaviendelangle is it a good time to include these components in the highlights? Or is it too early?

Reply:
On one side they are to early stage to be used in a real application so we should not advertise about them

On the other side, maybe it's better to be explicit and say "Hey they exist, but be careful" and to start to gather feedback from very early testers (but if we have too many they will be tons of feedbacks like "Mobile is broken" which we know)

Reply:
For now, we will get too many issues if users try to use it. Better to highlight this when we will have fix most of the issues we are already aware about.

Moreover we have no documentation, so they will struggle to use the component before realizing all the limitations